test/retcodes_test.py

```diff
@@ -170,3 +170,35 @@ def new_func(*args, **kwargs):
         with mock.patch('luigi.scheduler.Scheduler.add_task', new_func):
             self.run_and_expect('RequiringTask', 0)
             self.run_and_expect('RequiringTask --retcode-not-run 5', 5)
+
+    @with_config({'worker': {'keep_alive': 'True'}})
+    def test_retry_sucess_task(self):
+        class Bar(luigi.Task):
+            num = luigi.IntParameter()
+            has_run = False
+            run_count = 0
+
+            @property
+            def priority(self):
+                if self.num == 0:
+                    return 100
+                else:
+                    return 0
+
+            def run(self):
+                self.has_run = True
+                self.run_count += 1
+                if self.num == 0 and self.run_count == 1:
+                    raise ValueError()
+
+            def complete(self):
+                return self.has_run
+
+        class Foo(luigi.Task):
+            def requires(self):
+                for i in range(3):
```

Comment:
Can you skip requiring completely? Is only 1 task not enough? Like in the test with execution summary.